### DISCUSSION TITLE
Ruby-on-Rails ERB vs HAML

### DISCUSSION THREAD

**Discussion by bdavidxyz** [therubyradar/submissions#231] [Submissions]
This is an opinionated article about ERB and HAML with Rails. Let’s compare each other and pick a winner.
https://bootrails.com/blog/erb-vs-haml/

**Comment by Haraldwangsvik** [comment]
Interesting article, I personaly think HAML still is the better option. This because of how much more quickly you can create views using HAML rather than ERB
